src/PlusVolumeReconstruction/vtkPlusPasteSliceIntoVolume.cxx

```diff
@@ -733,6 +823,8 @@ const char* vtkPlusPasteSliceIntoVolume::GetCompoundingModeAsString( Compounding
     return "LATEST";
   case MEAN_COMPOUNDING_MODE:
     return "MEAN";
+  case IMPORTANCE_MASK_COMPOUNDING_MODE:
+	  return "IMPORTANCEMASK";
```

Comment:
The indentation is not the same. Style guildelines are available here
[https://app.assembla.com/spaces/plus/wiki/Developers_guide#coding_conventions](https://app.assembla.com/spaces/plus/wiki/Developers_guide#coding_conventions)